### ISSUE TITLE
One connection per request?

### ISSUE THREAD

**Issue by fengsp** [pallets-eco/flask-rq#4]
I have noticed that flask-rq does not use context-locals to save the redis connection, which means that everytime one request comes in, we are having a new connection.

What I need here is one connection manager so I do not have to handle connections myself.

Any reasons? So flask-rq is just a config manager? I do not see anything else meaningful here.

**Comment by joostdevries** [comment]
@fengsp @malthe I agree with you that using an app context would make sense. I plan to get some work done on this the coming days/weeks. Out of curiosity: are you using flask-script?

**Comment by italomaia** [comment]
Any news?

**Comment by davidism** [comment]
#44 completely rewrote the extension. Queues now share the `Redis` connection pool objects, which are created once during `init_app` rather than on each access.
